src/pages/home.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from 'styled-components';
+import imgItem1 from 'src/assets/img/imgItem1.svg';
+import section2Bg from 'src/assets/img/section2_bg.svg';
+import section2img from 'src/assets/img/section2img.png';
+import homeContent from '../../content/pages/home.yml';
+
+const Main = styled.main`
```

Comment:
I think you need to create Layout file, and wrap with it each page file. In this Layout file you can add background, header, footer etc.